app/views/unattended/provisioning_templates/registration/global_registration.erb

```diff
@@ -47,7 +47,7 @@ echo '#'
 echo '# Adding repository'
 echo '#'
 
-if [ x$ID = xrhel ] || [ x$ID = xfedora ] || [ x$ID = xol ] || test "${ID_LIKE#*rhel*}" != "$ID_LIKE" ; then
+if [ x$ID = xfedora ] || [ "${ID_LIKE#*fedora*}" != "$ID_LIKE" ]; then
```

Comment:
Just for my own sanity: https://github.com/theforeman/foreman/pull/8631#issuecomment-875422521 is the `os-release` for Oracle. Looks like this would work for that as well.

I do wonder if this replacement is POSIX shell compatible.

Reply:
Perhaps a lot easier, but why not:
```suggestion
if [ -f /etc/redhat-release ]; then
```

I checked that this file is present on Fedora, RHEL, CentOS, OracleLinux, AlmaLinux and RockyLinux.